### PR TITLE
phTrll 30

### PR DESCRIPTION
**Pull request by sanderPostma**
Notes:
- blockchainAgentHost is a new config param that needs to be added to the configmap
- I suppose REPO_API_HOST is overridden in the ConfigMap, otherwise it could never work. But needs to be tested to be sure.
- Still want to remove some temp console.log() commands as soon as I can get to it.

### CODE REVIEW

File: src/app.config.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "redirectUri": "/",
     "redirectUriLogout": "/logout"
   },
+  "blockchainAgentHost": "http://172.16.255.200:22240",
```
Comment (LDMGN) [inline src/app.config.json]: Can this be overridden from container parameters?

File: e2e/configs.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export const BROWSER_WAIT_TIMEOUT = 10000;
 export const USE_HASH_STRATEGY = true;
 
 // Repository configs
-export const REPO_API_HOST = 'https://triall.dev.sphereon.com';
+export const REPO_API_HOST = 'http://172.16.255.32';
```
Comment (LDMGN) [inline e2e/configs.ts]: You mention that this needs to be tested. How can this be tested? Has this been done already?

File: src/app/services/blockchain-proof/api/api/blockchain.service.ts [hunk 3]
```diff
@@ -11,97 +11,97 @@
...
+import {VerifyNodesResponse} from '../model/verifyNodesResponse';
 
-import { BASE_PATH, COLLECTION_FORMATS }                     from '../variables';
-import { Configuration }                                     from '../configuration';
+import {BlockchainAgentConfiguration} from '../blockchainAgentConfiguration';
+import {AppConfigService} from '@alfresco/adf-core';
 
 
 @Injectable()
 export class BlockchainService {
```
Comment (LDMGN) [inline src/app/services/blockchain-proof/api/api/blockchain.service.ts]: This class contains a lot of formatting. Please do formatting in a separate commit so we can clearly see the individual changes @sanderPostma

File: src/app.config.json (hunk 1 above)
Comment (sanderPostma) [reply src/app.config.json]: This whole file is in the ConfigMap

File: e2e/configs.ts (hunk 2 above)
Comment (sanderPostma) [reply e2e/configs.ts]: I suspect this file is only used when running in dev. It even has admin admin login in there and the end product is not using this otherwise it would never even be able login.

File: src/app/services/blockchain-proof/api/api/blockchain.service.ts (hunk 3 above)
Comment (sanderPostma) [reply src/app/services/blockchain-proof/api/api/blockchain.service.ts]: A lot? I have 7 refomatted lines above here. And the some imports actually changed also.
image

Comment (LDMGN) [reply src/app/services/blockchain-proof/api/api/blockchain.service.ts]: Yes, quite a lot. There are 158 changes of which most is only formatting.
